src/kaltura-player.ts

```diff
@@ -426,6 +426,10 @@ export class KalturaPlayer extends FakeEventTarget {
     return this._localPlayer.getActiveTracks();
   }
 
+  public changeChildQuality(track: any | string): void {
```

Comment:
what does the function do ? because it's not clear from the name

Reply:
mistake, removed.